www/src/scss/_theme-a-new-day.scss

```diff
@@ -2,6 +2,10 @@ body.a-new-day {
   background-color: #07ccf3;
   background-image: url("../../assets/img/custom_backgrounds/clouds.jpg");
 
+  .color-overlay {
+    background-color: #07ccf3;
```

Comment:
added color overlays to all themes ( as i was trying to support all themes initially) but ended up using only light-theme, dark-theme, high-contrast and low-light.

Reply:
Should we remove, as the `color-overlay` class seems to not be in use?

Reply:
it is used in the themes that currently support custom background. Initially, it was for all themes, so colour overlay existed everywhere. I can change it for now and add it later when we add support for more themes.